app/controllers/vacancies_controller.rb

```diff
@@ -25,6 +25,7 @@ def show
     vacancy = Vacancy.listed.friendly.find(id)
     return redirect_to(job_path(vacancy), status: :moved_permanently) if old_vacancy_path?(vacancy)
     @vacancy = VacancyPresenter.new(vacancy)
+    TrackVacancyPageViewJob.perform_later(vacancy.id) unless authenticated?
```

Comment:
I like this. This is also something that I imagine we're not doing with GA, as it's tricky for GA to know when someone is authenticated

Reply:
Super 👍 

It's not on the card (forgot) so I don't suggest we do it here but what we do get easily with GA is filtering known IP addresses out such as both offices. I imagine when we get to it we could easily extend this logic with the use of environment variables to hold some known IP addresses to check against the request IP, this would further improve the accuracy of this data and confident that it's real job seekers.